README.md

```diff
@@ -195,6 +196,16 @@ constructor(options) {
 - `session.secret`: The session secret. Set this to something unique.
 - `session.name`: The session name. Set this to something unique.
 
+### Global Configuration
+
+Any of the configuration parameters can be set globally for all bootstrap instances by calling `bootstrap.configure`. This is mostly relevant for running unit tests which will create a large number of bootstrap instances.
+
+```javascript
+const bootstrap = require('hof-bootstrap');
+// set the project root to a custom location for all bootstrap instances
+boostrap.configure('root', path.resolve(__dirname, '..'));
+```
+
```

Comment:
Thanks @lennym please add a link to Global Configuration from the menu at the top